### DISCUSSION TITLE
AI帶來的新危機? 射擊遊戲的新型態外掛

### DISCUSSION THREAD

**Discussion by neo910831** [1101-GS4719-C/Python-Community-GS4719-C#199] [閒聊八卦]
[警惕AI外挂！我写了一个枪枪爆头的视觉AI，又亲手“杀死”了它](https://www.youtube.com/watch?v=1gZ-KaWjhGY&t=124s)
  玩過射擊遊戲的人大多都知道，外掛的猖獗，嚴重破壞競技遊戲的平衡。遊戲公司往往得投注大量成本於「反外掛」上，對付這些竊取後台數據與竄改伺服器資料的「hacker」，然而隨著AI技術的進化，成本更低，破解難度更高的外掛，正在慢慢浮出水面。
  這部影片的作者，僅是透過手邊的人體骨架分析AI，套用到遊戲上，便能夠迅速且持續的將準星對準到頭部，達到過去外掛中「鎖頭」的效果。若是把AI的準星移動設定的更符合常理，屆時偵測難度將會大幅提升。若這樣的未來真的出現，開掛者將不再被稱為「Hacker 」，而是「Cheater」。因為這樣的作為更接近於讓AI幫忙代打。
  不過作者也提出了一個相應的解法，那便是用AI對付AI，也就是用AI去偵測準星移動的不合理性。要如何達到?這便是遊戲廠商得去苦惱的問題了。

**Comment by Tomlord1122** [comment]
其實遊戲公司有很多辦法可以知道玩家有沒有開外掛，像是以楓之谷為例，他會在你開遊戲前額外運行一個程式，在後台看檔案夾裡的資料有沒有被刻意竄改，藉此判斷外掛有沒有發生，當然還不只如此，像是紀錄玩家腳色的數據也是其中一種。但射擊遊戲的外掛可能就會比較難抓了，如果只是更改鼠標地走向，很難判斷他到底是不是外掛。

**Comment by wax75397** [comment]
感覺最後會是外掛AI會贏，若是外掛AI真的和一般人的動作相似的話，可能會出現大量誤抓的情況

**Comment by syt-110** [comment]
感覺外掛會減少遊戲品質增加遊玩成果，但如果加入AI的部分就沒有遊玩成果和體驗遊戲品質，雖然比較不會被抓，但是好像就沒有玩遊戲的必要了

**Comment by Isongzhe** [comment]
不管是什麼種類的外掛，一律都是 哭阿外掛死全家

**Comment by YuHengChen54** [comment]
用AI玩遊戲，那就不是自己在玩，那玩這個遊戲要幹嘛哈哈哈哈

**Reply by neo910831** [reply]
開掛者的心態不外乎是滿足虛榮心跟欺負人的快感，目的已經不只是玩遊戲了，大多的開掛者都是這心態。

**Comment by 109302013evantsao** [comment]
每次打APEX的時候遇到外掛都很討厭，感覺最近又比以前更多了，可能是這類AI外掛的興起導致的吧
